src/components/FirebaseAuth.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+
+import firebase from 'firebase';
+import StyleFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
+
+firebase.initializeApp({
+  apiKey: 'AIzaSyAXDqsWK4quNVaf9-YV2e28NsxkfA9rzJA',
+  authDomain: 'dreamerscholars.firebaseapp.com',
+});
```

Comment:
I think it'd be best to:
1. Refactor the config into a separate file. E.g. in `src/config/firebase.js` you could have this:
```js
export const firebaseConfig = {
  apiKey: 'AIzaSyAXDqsWK4quNVaf9-YV2e28NsxkfA9rzJA',
  authDomain: 'dreamerscholars.firebaseapp.com',
}
```
2. Import the config and initialize it in `src/App.jsx`:
```js
firebase.initializeApp(firebaseConfig);
```
We need to be able to use it in other places in the app besides auth.

Reply:
So I was able to successfully import from src/config/firebase.js and initialize it in the src/App.jsx using firebase.initializeApp(firebaseConfig); in between the function and return body. It worked compiled correctly but now the firebase auth login appears on the screen instead of as how I had it with the modal ill push the code for you to see on the Render PR